source/_integrations/alexa.smart_home.markdown

```diff
@@ -942,6 +943,19 @@ Requires [Proactive Events](#proactive-events) enabled.
 Intents to seek forwards (skip) or to rewind (go back) are not supported at the moment.
 {% endnote %}
 
+### Remote
+
+Supports changing the Remote `activity` from the given `activity_list`
+
+- _"Alexa, change the TV to PlayStation."_
+- _"Alexa, change the input on the TV to PlayStation."_
+
+{% note %}
+Alexa does not allow the following words to be used as activity names:
+
+`alarm`, `alarms`, `all alarms`, `away mode`, `bass`, `camera`, `date`, `date today`, `day`, `do not disturb`, `drop in`, `music`, `night light`, `notification`, `playing`, `sleep sounds`, `time`, `timer`, `today in music`, `treble`, `volume`, `way f. m.`
```

Comment:
Just because I am interested: How did you come up with this specific list?

Reply:
I got them from the alexa documentation. Those are reserved words and can be found here https://developer.amazon.com/en-US/docs/alexa/device-apis/resources-and-assets.html#reserved-words.